site-src/v1alpha2/references/policy-attachment.md

```diff
@@ -0,0 +1,290 @@
+# Policy Attachment
+
+Many implementations of the API support additional functionality that is not
+covered by the API directly. For example, concepts like timeouts, retries, or
+even CDN configuration are not sufficiently portable to be defined within the
+Gateway API. Instead, implementations may choose to represent these concepts
+with custom Policy resources.
+
+Policies attached to Gateway API resources and implementations must use the
+following approach to ensure consistency across implementations of the API.
+There are three primary components of this pattern:
```

Comment:
nit: dedup into a single sentence?

Reply:
The sentence before is pretty long, haven't found a great way to dedup yet.